cax/config.py

```diff
@@ -14,7 +14,7 @@
 
 PAX_DEPLOY_DIRS = {
     'midway-login1' : '/project/lgrandi/deployHQ/pax',
-    'tegner-login-1': '/afs/pdc.kth.se/projects/xenon/software/pax'
+    'tegner-login-1': '/afs/pdc.kth.se/projects/xenon/software/pax/pax_v4.10.2'
```

Comment:
Not necessarily, we can also use "pax_" + pax_version here. And if think about your comment in cax.json, it is maybe better. thx